spec/50-privacy.md

```diff
@@ -1,21 +1,23 @@
+The `traceparent` field has a predefined set of values. These values are randomly-generated numbers. If a random number generator has any logic of using user-identifiable information like IP address - this information may be exposed. Random number generators MUST NOT rely on any information that can potentially be user-identifiable.
 
-Another privacy risk of the `traceparent` field is an ability to correlate calls made as part of a single transaction. A downstream service may track and correlate two or more calls made in a single transaction and make assumptions about the identity of the caller of one call base on information in another call. Service initiating calls MAY choose to restart trace while making calls that might identify caller in the downstream service.
+Another privacy risk of the `traceparent` field is an ability to correlate calls made as part of a single transaction. A downstream service may track and correlate two or more calls made in a single transaction and make assumptions about the identity of the caller of one call base on information in another call. Service initiating calls MAY choose to restart a trace while making calls that might identify caller in the downstream service.
 
 Note, both privacy concerns of `traceparent` field are theoretical rather than practical.
 
 ## Privacy of tracestate field
 
-The field `tracestate` may contain any opaque value in any of the keys. The main purpose of this header is to provide additional information about the position of request in the multiple distributed tracing graphs.
+The  `tracestate` field may contain any opaque value in any of the keys. The main purpose of this header is to provide additional vendor-specific information to correlate traces across different software components. 
 
-Platforms and tracing systems MUST NOT include any personally identifiable information in the `tracestate` header.
+Tracing systems MUST NOT include any personally identifiable information in the `tracestate` header.
 
-Platforms and tracing systems extremely sensitive to personal information exposure MAY implement selective removal of values corresponding to the unknown keys. This mutation of the `tracestate` field is not forbidden, but highly discouraged. As it defeats the purpose of this field for allowing multiple tracing systems to collaborate.
\ No newline at end of file
+Platforms and tracing systems extremely sensitive to personal information exposure MAY implement selective removal of values corresponding to the unknown keys. This mutation of the `tracestate` field is not forbidden, but highly discouraged. As it defeats the purpose of this field for allowing multiple tracing systems to collaborate.
+
+*NOTE*: Should we restart the trace in this situation?
```

Comment:
what is the purpose of this note?

Reply:
the answer is no btw. there is a note that you may break other partners, but selective clean up of tracestate is not forcing you to break your own correlation

Reply:
I am not sure we agreed the answer is no. This means you would remove all keys except your own.  This was simply added to the spec without a thorough discussion